resources/zib/zib-MedicationAdministration2.xml

```diff
@@ -0,0 +1,318 @@
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/PractitionerRole" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Patient" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/RelatedPerson" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Device" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Patient" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-PractitionerRole" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-RelatedPerson" />
+      </type>
+      <mapping>
+        <identity value="MedicationAdministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.23380" />
+        <comment value="Patient" />
+      </mapping>
+      <mapping>
+        <identity value="MedicationAdministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.23172" />
+        <comment value="HealthProfessional" />
+      </mapping>
+      <mapping>
+        <identity value="MedicationAdministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.23355" />
+        <comment value="Caregiver" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.request">
+      <path value="MedicationAdministration.request" />
+      <short value="MedicationAgreement" />
+      <definition value="Medication agreement on which this administration is based." />
+      <alias value="Medicatieafspraak" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-MedicationAgreement" />
+      </type>
+      <mapping>
+        <identity value="MedicationAdministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.23170" />
+        <comment value="MedicationAgreement" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.note">
+      <path value="MedicationAdministration.note" />
+      <max value="1" />
+    </element>
+    <element id="MedicationAdministration.note.text">
+      <path value="MedicationAdministration.note.text" />
+      <short value="Comment" />
+      <definition value="Comments on administering the medication." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-medicationadministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.21337" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.dosage.route">
+      <path value="MedicationAdministration.dosage.route" />
+      <short value="RouteOfAdministration" />
+      <definition value="The route through which the medication is administered (oral, nasal, intravenous,...)." />
+      <alias value="Toedieningsweg" />
+      <binding>
+        <strength value="required" />
+        <description value="MedicatieToedieningToedieningswegCodelijst" />

```

Comment:
This can be deleted no descriptions for valuesets

Reply:
Removed!